src/main/java/org/zalando/nakadi/plugin/api/authz/EventAuthz.java

```diff
@@ -0,0 +1,9 @@
+package org.zalando.nakadi.plugin.api.authz;
+
+public interface EventAuthz
+{
+
+    String getCompatibilityMode();
```

Comment:
But I guess now that the names will conflict. I think you cannot have a method with the same name but two different return types.

Reply:
Fixed